app/models/submission.rb

```diff
@@ -143,6 +143,10 @@ def polish_language?
     self.language == POLISH
   end
 
+  def english_language
+    self.language == ENGLISH
+  end
```

Comment:
To nie wystarczy, tzn. trzeba dodać `scope` w `Submission`, który wybiera tylko artykuły w j. angielskim.